new-pipeline/dist.html

```diff
@@ -59,89 +59,92 @@ <h2 style="font-size: 18px; margin-bottom: 0">
-                                <dt title="Number of distinct dates shown">Number of dates</dt><dd id="prop-dates"></dd>
-                                <dt title="Range of dates currently shown">Selected Dates</dt><dd id="prop-date-range"></dd>
-                            </dl>
-                            <div class="scalar-only" style="margin-top: 50px">
-                                <dl class="dl-horizontal">
-                                    <dt>5th Percentile</dt><dd id="prop-p5"></dd>
-                                    <dt>25th Percentile</dt><dd id="prop-p25"></dd>
-                                    <dt>Median</dt><dd id="prop-p50"></dd>
-                                    <dt>75th Percentile</dt><dd id="prop-p75"></dd>
-                                    <dt>95th Percentile</dt><dd id="prop-p95"></dd>
-                                    <br>
-                                    <div style="font-size: 80%; margin: 0 20%">
-                                        <strong>Notice</strong> percentiles are estimated based on values in the histogram. Values are only guaranteed to be accurate to the nearest bucket.
-                                    </div>
-                                </dl>
-                            </div>
-                        </div>
-                        <div id="stats-comparison" style="margin-top: 50px; margin-bottom: 20px;">
-                            <table>
-                              <thead class="measure-headers">
-                              <tr>
-                              <th></th>
-                              <th title="5th percentile duration">5 %</th>
-                              <th title="25th percentile duration">25 %</th>
-                              <th title="Median duration">Median</th>
-                              <th title="75th percentile duration">75 %</th>
-                              <th title="95th percentile duration">95 %</th>
-                              </tr>
-                              </thead>
-                              <tbody id="histogram-percentiles" class="measure-values">
-                                  <tr>
-                                  <td colspan="6">No Data</td>
-                                  </tr>
-                              </tbody>
-                            </table>
-                        </div>
-                        <div style="text-align: center">
-                            <a class="btn btn-default btn-sm" id="export-csv" title="Export the current histogram to CSV">Export CSV</a>
-                            <a class="btn btn-default btn-sm" id="export-json" title="Export the current histogram to JSON">Export JSON</a>
-                        </div>
-                    </div>
-                </figure>
+                  <div>
+                    <figcaption style="text-align: center" id="use-counter-link"></figcaption>
```

Comment:
I think it would be better to place the link after the `caption-div`.
That way we put the description first, then show additional helpful links after it.

The `style` here should go into a CSS file.

This link, when shown, still has a different style from e.g. `dist-caption-link` (smaller font size & blue text color instead of black).
Can we give it the same style?